static/index.html

```diff
@@ -11,25 +11,34 @@
       <h1 class="title">Afterlight Caves</h1>
       <span id="version">Beta version 0.6.3</span>
 
-      <input
-        type="text"
-        id="name-input"
-        maxlength="64"
-        placeholder="Name (optional)"
-      />
-
       <div id="gamediv"></div>
 
-      <p style="text-align:center">
-        🎮🎮🎮 <b>Try using a controller for analog controls!</b> 🎮🎮🎮<br />
-        wasd <i>(or left sick)</i>: move<br />
-        arrow keys <i>(or right stick)</i>: shoot<br />
-        space <i>(or LB/L1)</i>: bomb<br />
-        space <i>(or A/cross)</i>: select<br />
-        tab <i>(or B/circle)</i>: back<br />
-        f: fullscreen<br />
-        p <i>(or start)</i>: pause<br />
+      <p>
+        <b>Try using a controller for analog controls!</b><br />
       </p>
+      <table>
+        <tr>
+          <td>move</td> <td>wasd</td> <td>left stick</td>
```

Comment:
Can you have the right-most column right justified?

Reply:
Implemented in 706c426